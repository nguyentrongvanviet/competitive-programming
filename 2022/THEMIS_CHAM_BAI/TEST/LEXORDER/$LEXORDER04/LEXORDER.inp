9 21412
6 8 7 5 3 9 1 4 2 