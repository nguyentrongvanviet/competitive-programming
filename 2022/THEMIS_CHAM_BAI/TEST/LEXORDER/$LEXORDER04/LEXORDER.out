7 4 1 5 9 8 6 3 2 